metrics/api/src/main/java/io/helidon/metrics/api/Clock.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.metrics.api;
+
+/**
+ * Reports absolute time (and, therefore, is also useful in computing elapsed times).
+ */
+public interface Clock extends Wrapped {
+
+    /**
+     * Returns the current wall time in milliseconds since the epoch.
+     *
+     * <p>
+     *     Typically equivalent to System.currentTimeMillis. Should not be used to determine durations.
+     *     For that use {@link #monotonicTime()} instead.
+     * </p>
+     *
+     * @return wall time in milliseconds
+     */
+    long wallTime();
```

Comment:
Maybe `Instant` would be a better return type here? Is there a reason we need milliseconds?